src/borg/archiver.py

```diff
@@ -2431,34 +2429,34 @@ def do_break_lock(self, args, repository):
             P sh
             R /
             # can be rebuild
-            - /home/*/.cache
+            - home/*/.cache
             # they're downloads for a reason
-            - /home/*/Downloads
+            - home/*/Downloads
             # susan is a nice person
             # include susans home
-            + /home/susan
+            + home/susan
             # also back up this exact file
-            + pf:/home/bobby/specialfile.txt
+            + pf:home/bobby/specialfile.txt
             # don't backup the other home directories
-            - /home/*
+            - home/*
             # don't even look in /proc
-            ! /proc
+            ! proc
 
         You can specify recursion roots either on the command line or in a patternfile::
 
             # these two commands do the same thing
-            borg create --exclude /home/bobby/junk repo::arch /home/bobby /home/susan
+            borg create --exclude home/bobby/junk repo::arch /home/bobby /home/susan
             borg create --patterns-from patternfile.lst repo::arch
 
         The patternfile::
 
             # note that excludes use fm: by default and patternfiles use sh: by default.
             # therefore, we need to specify fm: to have the same exact behavior.
             P fm
-            R /home/bobby
-            R /home/susan
+            R home/bobby
+            R home/susan
```

Comment:
oops. guess i missed that in the last review. this is not a pattern, but a recursion root, guess it must not be changed.

Reply:
Shall I fix that in master branch, too?

Reply:
yes, please.

Reply:
Fixed & new PR for master.